pandas/tests/groupby/transform/test_transform.py

```diff
@@ -418,45 +413,36 @@ def test_transform_select_columns(df):
     tm.assert_frame_equal(result, expected)
 
 
-@pytest.mark.parametrize("duplicates", [True, False])
-def test_transform_exclude_nuisance(df, duplicates):
```

Comment:
In order to still hit the warning this was testing for, I needed to switch from a duplicated float column to a duplicated string column, as we no longer warn in the case of the duplicated float column. But using a string column would always fail with SeriesGroupBy, which is why duplicates=False was removed.